test-data/unit/check-attr.test

```diff
@@ -428,6 +428,38 @@ a = A.new()
 reveal_type(a.foo) # E: Revealed type is 'def () -> builtins.int'
 [builtins fixtures/classmethod.pyi]
 
+[case testAttrsOtherOverloads]
+import attr
+from typing import overload, Union
+
+@attr.s
+class A:
+    a: int
+    b: str
```

Comment:
I would rather make these `attr.ib`s, to check that the `cls` signature below is generated correctly by the plugin.

Reply:
Also done